addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderThingMonitorHandler.java

```diff
@@ -838,7 +838,7 @@ protected State getDetailedStatus() {
             }
 
         } catch (Exception ex) {
-            logger.debug(ex.getMessage());
+            logger.debug("{}", ex.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace